CODE_OF_CONDUCT.md

```diff
@@ -75,11 +76,11 @@ spaces:
 -   **Be considerate.** Your work will be used by other people, and you in turn
     will depend on the work of others. Any decision you make will affect users
     and colleagues, and you should take those consequences into account.
-    Remember that we’re a world-wide community, so you might not be
-    communicating in someone else’s primary language.
+    Remember that we’re a worldwide community, so you might not be communicating
+    in someone else’s primary language.
 
 -   **Be kind.** Not all of us will agree all the time, but disagreement is no
-    excuse for poor behavior and hurtful words. We might all experience some
+    excuse for poor behaviour and hurtful words. We might all experience some
```

Comment:
I am still seeing spelling changes for "behavior".  Did you miss a `git stash pop` or something?

Reply:
Yes please.